portal-impl/src/com/liferay/portal/verify/VerifyProperties.java

```diff
@@ -41,9 +42,17 @@ public class VerifyProperties extends VerifyProcess {
 	protected void doVerify() throws Exception {
 		verifySystemProperties();
 
-		verifyPortalProperties();
+		List<String> keys = verifyPortalProperties();
 
 		verifyDocumentLibrary();
+
+		if (!keys.isEmpty()) {
+			System.out.println(
```

Comment:
@W2jx You can use `_log` here.

Reply:
I will fix it. The reason why I use println is that I thought a log like this would be more objective.